src/__tests__/cat.test.ts

```diff
@@ -225,9 +314,208 @@ for (const format of ['symbolic', 'semantic'] as Array<'symbolic' | 'semantic'>)
+    const cat = new Cat({ minTheta: -1, maxTheta: 1, priorDist: 'norm', priorPar: [0, 1] });
+    const priorXValues = cat.prior.map((p) => p[0]);
+    
+    // Check that steps are approximately 0.1 apart
+    for (let i = 1; i < priorXValues.length; i++) {
+      const step = priorXValues[i] - priorXValues[i-1];
+      expect(step).toBeCloseTo(0.1, 6); // 6 decimal places due to rounding fix
+    }
+  });
+
+  it('should handle edge case with very small stepSize', () => {
+    const cat = new Cat({ minTheta: 0, maxTheta: 1, priorDist: 'norm', priorPar: [0.5, 0.1] });
+    expect(cat.prior.length).toBeGreaterThan(1);
+    expect(cat.prior[0][0]).toBeCloseTo(0, 6);
+    expect(cat.prior[cat.prior.length - 1][0]).toBeCloseTo(1, 6);
+  });
+
+  it('should create uniform prior distribution', () => {
+    const cat = new Cat({ priorDist: 'unif', priorPar: [-2, 2], minTheta: -3, maxTheta: 3 });
+    expect(cat.priorDist).toBe('unif');
+    
+    // Check that all probabilities are equal (uniform distribution)
+    const probs = cat.prior.map(([, p]: [number, number]) => p);
+    const xs = cat.prior.map(([x]: [number, number]) => x);
+    
+    // Find points within the uniform bounds [-2, 2]
+    const probsInBounds = probs.filter((p, i) => xs[i] >= -2 && xs[i] <= 2);
+    
+    // All points within bounds should have the same probability
+    const uniformProb = probsInBounds[0];
+    probsInBounds.forEach(prob => {
+      expect(prob).toBeCloseTo(uniformProb, 6);
+    });
+    
+    // Points outside bounds should have zero probability
+    const probsOutsideBounds = probs.filter((p, i) => xs[i] < -2 || xs[i] > 2);
+    probsOutsideBounds.forEach(prob => {
+      expect(prob).toBeCloseTo(0, 6);
+    });
+    
+    // Check that the total probability sums to 1
+    const totalProb = probs.reduce((sum, prob) => sum + prob, 0);
+    expect(totalProb).toBeCloseTo(1, 6);
+    
+    // Check that the range is correct (should use priorPar bounds, not minTheta/maxTheta)
+    expect(Math.min(...xs)).toBeCloseTo(-3, 6);
+    expect(Math.max(...xs)).toBeCloseTo(3, 6);
+  });
+    
+
+  it('should use default priorPar for uniform distribution', () => {
+    const cat = new Cat({ priorDist: 'unif' });
+    expect(cat.priorDist).toBe('unif');
+    expect(cat.priorPar).toEqual([-4, 4]);
+    expect(cat.prior.length).toBeGreaterThan(0);
+  });
+
+  it('should use fallback normal(0,1) when priorDist is invalid', () => {
+    // This test covers the fallback branch: normal(0, 1, minTheta, maxTheta)
+    // We need to test this by temporarily modifying the validation to allow invalid priorDist
+    const originalValidatePrior = Cat['validatePrior'];
+    
+    // Temporarily replace validatePrior to allow invalid priorDist
+    Cat['validatePrior'] = () => {
+      // No-op validation to allow invalid priorDist
+    };
+    
+    try {
+      const cat = new Cat({ 
+        minTheta: -2, 
+        maxTheta: 2,
+        priorDist: 'invalid' as any,
```

Comment:
I recommended some changes in `cat.ts` that would obviate this entire test. I think we should not support falling back to `normal` if `priorDist` is invalid. It should just throw an error and not let the user move on from there.